pkgs/tools/networking/iperf3d/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "iperf3d";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "wobcom";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-pMwGoBgFRVY+H51k+YCamzHgBoaJVwEVqY0CvMPvE0w=";
+  };
+
```

Comment:
Shouldn't `iperf3` be a dependency of `iperf3d`. I't does not work otherwise anyway.

Reply:
I have now added a wrapper which ensures that `iperf3` is in the `PATH` of `iperf3d`. Can you please try it again and confirm that this fix works for you?